tests/unit/ext/auth/database.py

```diff
@@ -15,6 +13,18 @@
 
 
 @pytest.fixture
+def dbauth():
+    from haas.ext.auth import database
```

Comment:
What is the benefit of having this here instead of at the top as a module level import?